submissions/Sergey-Kali/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/index.css">
+    <title>My menu</title>
+</head>
+<body>
+   <nav class="nav-panel">
+    <ul class="nav-list">
+        <li class="list-item"><a href="#" class="mail">Mail</a></li>
+        <li class="list-item"><a href="#" class="img">Images</a></li>
+        <li class="list-item">
+            <div class="popup-wrapper">
+                <input type="checkbox" id="popup-button" class="popup-input">
+                <label for="popup-button" class="popup-label"></label>
+                <div class="popup-menu">
+                    <div class="div-for-menu">
+                        <a href="#" class="popup-item"><img src="/images/calendar.png" alt="Calendar">
+                            <p>Calendar</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/mail.png" alt="Mail">
+                            <p>Mail</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/news.png" alt="News">
+                            <p>News</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/maps.png" alt="Maps">
+                            <p>Maps</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/files.png" alt="Files">
+                            <p>Files</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/contacts.png" alt="Contacts">
+                            <p>Contacts</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/tanslation.png" alt="Translate">
+                            <p>Translate</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/videos.png" alt="Videos">
```

Comment:
Fix aspect ratio of icons. [Read about this](https://jakearchibald.com/2022/img-aspect-ratio/)
![image](https://user-images.githubusercontent.com/16196199/187024270-22872bb6-260b-462f-b019-6bace938de72.png)